File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/Configuration/MaxRequests.cs
using Raven.Client.Documents;

namespace Raven.Documentation.Samples.ClientApi.Session.Configuration
{
    public class MaxRequests
    {
        public MaxRequests()
        {
            using (var store = new DocumentStore())
            {
                using (var session = store.OpenSession())
                {
                    #region max_requests_1
                    session.Advanced.MaxNumberOfRequestsPerSession = 50;
                    #endregion
                }

                #region max_requests_2
                store.Conventions.MaxNumberOfRequestsPerSession = 100;
                #endregion
            }
        }
    }
}

File: Documentation/4.0/Samples/nodejs/client-api/session/configuration/maxRequests.js
import { DocumentStore } from "ravendb";

const store = new DocumentStore();

{
    const session = store.openSession();
    //region max_requests_1
    session.advanced.maxNumberOfRequestsPerSession = 50;
    //endregion

    //region max_requests_2
    store.conventions.maxNumberOfRequestsPerSession = 100;
    //endregion
}
